<launch>
	<node name = "controller_node"
    		pkg = "ros_robotics"
    		type = "controller_train5_1"		
    		/>

	<!--node name = "speed_control_node"
    		pkg = "ros_robotics"
    		type = "speed_control_train5"
    		/-->

	<!--node name = "energy_node"
    		pkg = "ros_robotics"
    		type = "energy_train5"
    		/-->

	<node name = "Xt_handler_node"
    		pkg = "ros_robotics"
    		type = "Xt_train5.py"
    		/>

	<node name = "terrain_model_node"
    		pkg = "ros_robotics"
    		type = "terrain_model_train5"
    		output="screen"/>

	<node name = "motor_model_node"
    		pkg = "ros_robotics"
    		type = "motor_model_train5"
    		/>




</launch>
